include/groonga/db.h

```diff
@@ -42,6 +42,24 @@ GRN_API void
 grn_db_touch(grn_ctx *ctx, grn_obj *db);
 GRN_API grn_rc
 grn_db_recover(grn_ctx *ctx, grn_obj *db);
+/**
+ * \brief Unmaps all opened tables and columns in the passed database.
+ *        Resources used by these opened tables and columns are freed.
+ *        Normally, this API is needless. Because resources used by opened
+ *        tables and columns are managed by OS automatically.
+ *
+ * \note This is an experimental API.
```

Comment:
Is it still an experimental API? Maybe we can remove this one in this change.

Reply:
We can mark this as a stable API.